libs/common/grpc.go

```diff
@@ -47,6 +48,11 @@ func StartNewGRPCServer(addr string, registerServerHook func(*grpc.Server)) {
 
 	registerServerHook(server)
 
+	if hwutil.MatchEnv("MODE", DevelopmentMode) {
```

Comment:
you should use [`common.Mode`](https://github.com/helpwave/services/blob/main/libs/common/setup.go#L24) instead